doc/classes/EditorNode3DGizmoPlugin.xml

```diff
@@ -59,8 +59,11 @@
 			</argument>
 			<argument index="1" name="billboard" type="bool" default="false">
 			</argument>
+			<argument index="2" name="icon" type="Texture" default="null">
```

Comment:
Did you add this manually? The type should be `Texture2D` and it should be inferred properly by `doctool` normally,  unless there's a bug (https://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html#updating-the-documentation-template).
`Texture` in the `master` branch is now an abstract class for all types of textures (including `Texture3D`).

Reply:
Oops, I copied it from my 3.2 patch.